resources/fetchers/iam_fetcher.go

```diff
@@ -46,25 +46,36 @@ type IAMResource struct {
 
 func (f IAMFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
 	f.log.Debug("Starting IAMFetcher.Fetch")
+	iamResources := make([]awslib.AwsResource, 0)
 
 	pwdPolicy, err := f.iamProvider.GetPasswordPolicy(ctx)
 	if err != nil {
-		return err
+		f.log.Errorf("Unable to fetch PasswordPolicy, error: %v", err)
+	} else {
+		iamResources = append(iamResources, pwdPolicy)
```

Comment:
how will logging the error instead of returning it will affect the flow? is this by design?

Reply:
In case we were able to fetch a certain resource but fail to get the other, IMO, a better experience will be to generate some findings rather than nothing.

We have similar behavior in the `file system fetcher`.